assets.md

```diff
@@ -10,6 +10,32 @@ This file is intended to list all the assets controlled by OpenTelemetry.
   - PR: https://github.com/1Password/1password-teams-open-source/pull/364
   - Admins: [@open-telemetry/governance-committee](https://github.com/orgs/open-telemetry/teams/governance-committee)
 
+## Community Resource Accounts
+
+### AWS account
+
+Link: http://cncf-aws-opentelemetry.signin.aws.amazon.com/
+
+- Community account to published Lambda layers
+- Admin: Alex Boten @codeboten & Anthony Mirabella @aneurysm9
```

Comment:
@open-telemetry/lambda-extension-maintainers could you request access to these as well?

Reply:
How do we do so?